theme/tutorial.less

```
@@ -394,6 +394,7 @@ code.lang-filterblocks {
 .tutorial-callout > div {
     max-height: 60vh;
     overflow: auto;
+    padding-right: 3vh;
```

```suggestion
    padding-right: 2rem;
```

Thinking about it again probably want to use em instead of vh -- vh changes with the browser window (well, viewport) height so this would grow / shrink based off the height of the browser window, when we really just want it to be 'a bit larger so the 1.5rem button never touches this'